src/renderer/components/status-bar/status-bar.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import styles from "./status-bar.module.scss";
+
+import React from "react";
+import { observer } from "mobx-react";
+import { withInjectables } from "@ogre-tools/injectable-react";
+import registeredStatusBarItemsInjectable, { RegisteredStatusBarItems } from "./registered-status-bar-items.injectable";
+import type { IComputedValue } from "mobx";
+
+export interface StatusBarProps {}
+
+interface Dependencies {
+  items: IComputedValue<RegisteredStatusBarItems>
+}
+
+const NonInjectedStatusBar = observer(({ items }: Dependencies & StatusBarProps) => {
+  const { left, right } = items.get();
+
+  return (
+    <div className={styles.StatusBar}>
+      <div className={styles.leftSide}>
+        {left.map((Item, index) => (
+          <div className={styles.item} key={index}>
+            <Item />
+          </div>
+        ))}
+      </div>
+      <div className={styles.rightSide}>
+        {right.map((Item, index) => (
+          <div className={styles.item} key={index}>
+            <Item />
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+
+});
```

Comment:
@jim-docker I mostly just renamed some files, but the main part of the bug fix is here and in `src/renderer/components/status-bar/registered-status-bar-items.injectable.tsx`.

The bug was occurring because the old implementation assumed that there was always going to be an item on the left hand side (the active hotbar name). However, that was removed in #4679 which caused this bug to be noticed.

The solution that I have done here is to actually separate the left and the right sides in the DOM (that is what the next `div`'s are for) and use CSS to spread out those. Instead of spreading out the items themselves.